src/azul/__init__.py

```diff
@@ -140,6 +141,20 @@ def data_portal_name(self):
     def dss_endpoint(self) -> str:
         return os.environ['AZUL_DSS_ENDPOINT']
 
+    @property
+    def tdr_bigquery_dataset(self) -> BigQueryDataset:
```

Comment:
Add `@property def tdr_dataset(self) -> str:`

for just the env var access.

Reply:
Implementing as `@property def tdr_target(self) -> str:` 
See my comments above